data_import/utils/import_utility.py

```diff
@@ -55,18 +54,26 @@ def select_unseen_responding_agency(self):
                 q.add({'name': agency[0]})
 
     def insert_responding_agency(self):
-        insert = '''
-            INSERT INTO data_import_respondingagency (name)
-              SELECT
-                DISTINCT responding_agency
-              FROM {raw_payroll} AS raw
-              LEFT JOIN data_import_respondingagency AS existing
-              ON TRIM(LOWER(raw.responding_agency)) = TRIM(LOWER(existing.name))
-              WHERE existing.name IS NULL
+        # Unseen names mapped to an existing responding agency will have been
+        # added as aliases, i.e., they will no longer appear in this select.
+        from data_import.models import RespondingAgency, RespondingAgencyAlias
```

Comment:
I imported models in the functions that use them in this file because otherwise, there are some circular imports that I don't have time to track down.